hieradata/development.yaml

```diff
@@ -196,6 +196,8 @@ govuk_containers::app::config::global_envvars:
 
 govuk_elasticsearch::version: '2.4.6'
 govuk_elasticsearch::cors_enabled: true
+govuk_elasticsearch::repo_path:
```

Comment:
Might be less confusing to name this something like `snapshot_path`? I know Elasticsearch uses `path.repo`, but "repository" will make people think of "code repository" rather than "DB backup".

Reply:
I think it makes sense to stay inline with the field it is used for. If people are confused/looking at it we can change it later.